Source/module.modulemap

```diff
@@ -0,0 +1,29 @@
+module OCMock {
```

Comment:
in my experience, in many cases making this `framework module OCMock` is the canonical way of describing this to allow framework header or modular import.  (though that is not what we did in the example i describe below of a legacy bit of code that we added module.modulemap support to …)

Reply:
With `framework module OCMock`, I see:

```
$ swift test
/Users/paulbeusterien/gh/ocmock/Source/module.modulemap:2:10: error: header 'OCMock/OCMock.h' not found
  header "OCMock/OCMock.h"
         ^
/Users/paulbeusterien/gh/ocmock/SwiftPMTests/objc/objc-module.m:1:9: fatal error: could not build module 'OCMock'
@import OCMock;
 ~~~~~~~^~~~~~
2 errors generated.
```

Reply:
in that case, the header should possibly simply be

`    header "OCMock.h"`

… however, it may also be the case that if you follow the guide of the other example i provided, `framework` may not be necessary.

Reply:
Changing to `header "OCMock.h"` leads to `fatal error: 'OCMock.h' file not found`

It seems functional to add the wrapper scope like in the example, but I don't see what value it provides. I don't see any different functionality.

Reply:
the wrapper scope in the example allows for submodule import, should that be desired.

we do perform submodule import of items in the example.  when performing `@import OCMock` in our current code, we cannot perform submodule import, so have never done so.  i don't think it's necessary, since nothing is really relying on it, and since for most use cases, it is test code, and worrying about eyeball complexity based on which submodules you are importing probably isn't as important.  i can live without the submodule wrapper separation.

i am surprised that the `header "OCMock/OCMock.h"` syntax is fully required.  that has not been something we have encountered in creating module.modulemap files.

if this works for the OCMock tests and works in modules you ingesting OCMock in, this feels like an appropriate start.

Reply:
As I mentioned before I don't know Swift Package Manager. This exchange leaves me puzzled. Should it be `framework`, `module`, and/or `header`? Have you reached consensus? 

Also, please bear in mind that I do not want to change the framework header. So, anything that requires double quotes or similar in OCMock's public header files is a non-starter.